packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -2872,22 +3069,35 @@ A Custom Role is a custom set of [Permissions](#permission-types). A Custom Role
     "id": "cr0Yq6IJxGIr0ouum0g3",
     "label": "UserCreator",
     "description": "Create users",
-    "permissions": [
-      "okta.users.create",
-      "okta.users.read",
-      "okta.groups.read",
-      "okta.users.profile.manage"
-    ]
+    "created": "2021-02-06T16:20:57.000Z",
+    "lastUpdated": "2021-02-06T16:20:57.000Z",
+    "_links": {
+    "permissions": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3/permissions"
+    },
+    "self": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3"
+    }
+  }    
 }
 ```
 ### Permission types
 
 <ApiLifecycle access="beta" />
 
-Permissions can be used to build Custom Roles. Permissions to manage a resource also grant the viewing privileges for the same resource so that you won't need to assign them separately.
+Permissions can be added to, or removed from, custom Roles.  Permissions to manage a resource also grant the viewing privileges for the same resource so that you won't need to assign them separately.
 
 User permissions are only effective with respect to the Group(s) to which the admin is granted a Role through Resource Set assignments.
 
+### Permission properties
```

Comment:
Are these permission properties or role properties? I think it's a bit confusing to attribute timestamp to permission. We should at least clarify whether `created` and `lastUpdated` refer to the timestamp of the role-permision association or the role

Reply:
I'll also make this a #### - i.e. a subsection of Permission types for clarity